test/Passes/DefaultPipeline/default-tpp-passes.mlir

```diff
@@ -219,16 +219,12 @@ func.func @softmax(%arg0: tensor<2x2x2x2xf32>, %arg1: tensor<2x2x2x2xf32>) -> te
 // CHECK-LABEL: batch_matmul_rewrite
 func.func @batch_matmul_rewrite(%arg0: tensor<512x32x64xf32>, %arg1: tensor<512x64x32xf32>) -> tensor<512x32x32xf32> {
   %0 = tensor.empty() : tensor<512x32x32xf32>
-  // CHECK-DAG: %[[C1:.+]] = arith.constant 1 : i64
-  // CHECK-DAG: %[[C32:.+]] = arith.constant 32 : i64
-  // CHECK-DAG: %[[C64:.+]] = arith.constant 64 : i64
-  // CHECK-DAG: %[[C0:.+]] = arith.constant 0 : i64
   // CHECK-DAG: %[[C0_i:.+]] = arith.constant 0 : index
   // CHECK-DAG: %[[C1_i:.+]] = arith.constant 1 : index
   // CHECK-DAG: %[[C512_i:.+]] = arith.constant 512 : index
-  // CHECK: %{{.+}} = call @xsmm_gemm_dispatch(%[[C1]], %[[C32]], %[[C32]], %[[C64]], %[[C64]], %[[C32]], %[[C32]], %[[C0]])
+  // CHECK: %{{.+}} = call @xsmm_brgemm_dispatch
```

Comment:
What are the actual dimensions?

Reply:
I don't think we should care about such details in a generic pipeline test.
It just ensures things work together. That's sth for a specific pass test.